docs/src/install/index.md

```diff
@@ -4,200 +4,213 @@ tags: [Installation]
 
-:::note
-Some OSes may require you to use `sudo` to install packages.
+:::tip
+If you are having problems setting any of the repositories, check out the [Cloudsmith](https://cloudsmith.io/~thinedge/repos/tedge-release/setup/#formats-deb) website where they have **Set Me Up** instructions in additional formats, e.g. manual configuration rather than via the `setup.*.sh` script.
 :::
 
-### Download package
+**Pre-requisites**
 
-thin-edge.io package is in thin-edge.io repository on GitHub: [thin-edge.io](https://github.com/thin-edge/thin-edge.io/releases).
+The instructions require you to have the following tools installed.
 
-To download the package from github repository use following command (use desired version):
+* curl
+* bash
 
-```sh
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/<package>_<version>_<arch>.deb
-```
+#### Setup
 
-Where:
-- `version` -> thin-edge.io version in x.x.x format
-- `arch` -> architecture type (amd64, armhf, arm64)
+**Running with sudo**
 
-```sh title="Example"
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.9.0/tedge_0.9.0_armhf.deb
-```
-
-and for `mapper`:
+You will need to have `sudo` also installed if you want to run these instructions.
 
-```sh title="Example"
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.9.0/tedge-mapper_0.9.0_armhf.deb
+```sh tab={"label":"Debian/Ubuntu"}
+curl -1sLf 'https://dl.cloudsmith.io/public/thinedge/tedge-main/setup.deb.sh' | sudo bash
 ```
 
-### Install downloaded package
-
-Now, we have downloaded the package we can proceed to installation. First we will install cli tool `tedge`.
+```sh tab={"label":"RHEL/Fedora/RockyLinux"}
+curl -1sLf 'https://dl.cloudsmith.io/public/thinedge/tedge-main/setup.rpm.sh' | sudo bash
+```
 
-:::note
-Some OSes may require you to use `sudo` to install packages and therefore all following commands may need `sudo`.
-:::
+```sh tab={"label":"Alpine"}
+curl -1sLf 'https://dl.cloudsmith.io/public/thinedge/tedge-main/setup.alpine.sh' | sudo bash
+```
 
-To install `tedge` use following command:
+**Running as root**
 
-```sh
-sudo dpkg -i tedge_<version>_<arch>.deb
-```
+These commands must be run as the root user.
 
-```sh title="Example"
-sudo dpkg -i tedge_0.5.0_armhf.deb
+```sh tab={"label":"Debian/Ubuntu"}
+curl -1sLf 'https://dl.cloudsmith.io/public/thinedge/tedge-main/setup.deb.sh' | bash
 ```
 
-To install mapper for thin-edge.io do:
-
-```sh
-sudo dpkg -i tedge-mapper_<version>_<arch>.deb
+```sh tab={"label":"RHEL/Fedora/RockyLinux"}
+curl -1sLf 'https://dl.cloudsmith.io/public/thinedge/tedge-main/setup.rpm.sh' | bash
 ```
 
-```sh title="Example"
-sudo dpkg -i tedge-mapper_0.9.0_armhf.deb
+```sh tab={"label":"Alpine"}
+curl -1sLf 'https://dl.cloudsmith.io/public/thinedge/tedge-main/setup.alpine.sh' | bash
 ```
 
-## Uninstall
```

Comment:
Is this moved to some other doc?

Reply:
I'll add a new uninstall guide in a follow up PR, as this has also changed. But more changes are needed, e.g. changing the script, or instructions.

We can do this work in a followup PR because user's generally don't run the uninstall very often, so just focussing on the install part makes more sense as a first step.